### ISSUE TITLE
Trivy Scanner showing CVE's in v1.20.0 image, any plan to patch these? 

### ISSUE THREAD

**Issue by Maven35** [GoogleContainerTools/kaniko#2992]
i have taken scans using Trivy and its reporting a few CVE's. is there currently any plan to update these and patch these open findings? 

```
kaniko/executor (gobinary)
==========================
Total: 5 (UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 2, CRITICAL: 2)
```

thanks in advance for looking  at this issue. if you need me to post more specific information i can.

**Comment by dutchhaag** [comment]
Running into the same issue. Here is a more in-depth view of the CVEs / Libs.

| Lib                            | CVEs                                         | Installed Version | Fixed Version | Links                                                                                                                            |
|--------------------------------|----------------------------------------------|-------------------|---------------|----------------------------------------------------------------------------------------------------------------------------------|
| github.com/moby/buildkit       | CVE-2024-23651 CVE-2024-23652 CVE-2024-23653 | v0.11.6           | v0.12.5       | https://avd.aquasec.com/nvd/cve-2024-23651 https://avd.aquasec.com/nvd/cve-2024-23652 https://avd.aquasec.com/nvd/cve-2024-23653 |
| github.com/opencontainers/runc | CVE-2024-21626                               | v.1.1.5           | v1.1.12       | https://avd.aquasec.com/nvd/cve-2024-21626                                                                                       |
|                                |                                              |                   |               |                                                                                                                                  |

**Comment by aaron-prindle** [comment]
This has been patched now via #2995 which is present in the latest release `v1.20.01`, scan below by `grype`:
```
aprindle@aprindle-ssd ~/kaniko  [main]grype gcr.io/kaniko-project/executor:latest
 ✔ Vulnerability DB        [no update available]
New version of grype is available: 0.74.5 (currently running: 0.61.0)
 ✔ Loaded image            
 ✔ Parsed image            
 ✔ Cataloged packages      [211 packages]
 ✔ Scanning image...       [2 vulnerabilities]
   ├── 0 critical, 1 high, 1 medium, 0 low, 0 negligible
   └── 0 fixed
No vulnerabilities found
```

PRs are welcome for any such vulns in the future, thanks

**Comment by remy-tiitre** [comment]
It seems that CVE-2024-21626 is back in 1.21.0 version

**Comment by aaron-prindle** [comment]
This is fixed @ HEAD now with https://github.com/GoogleContainerTools/kaniko/pull/3038 merged
```
aprindle@aprindle-ssd ~/kaniko  [main]grype gcr.io/kaniko-project/executor:main
 ✔ Vulnerability DB        [no update available]
New version of grype is available: 0.74.7 (currently running: 0.61.0)
 ✔ Loaded image            
 ✔ Parsed image            
 ✔ Cataloged packages      [211 packages]
 ✔ Scanning image...       [2 vulnerabilities]
   ├── 0 critical, 1 high, 1 medium, 0 low, 0 negligible
   └── 0 fixed
No vulnerabilities found
```

I plan on creating a patch release this week to resolve this, v1.21.1
